### ISSUE TITLE
Horizontal Stacked Bar with first bar value 0 (zero) issue

### ISSUE THREAD

**Issue by GalakFayyar** [FVANCOP/ChartNew.js#268]
Hello,

First of all, I really appreciate the library you produce for graphics and information display technics. It's a really great job and it helps a lots developments and creativity. I hope it will continue in this way.

However, I have some trouble using HorizontalStackedBar... I carefully read some issues (#228 and #205) but I can't make it works ! Everytime I put in my horizontal stacked bar a first element with a value equals to 0 (zero), the following stacked bar is not display. I still got the label and the value, but the bar is invisible.
It appears to be working on classic stacked bars (vertical), but not for horizontal ones :

![image](https://cloud.githubusercontent.com/assets/11835291/7023307/8339df16-dd34-11e4-8555-3758915268ef.png)

Can you help me in solving this ?

**Comment by FVANCOP** [comment]
Hi,

Sorry for the long delay - I was disconnected for a week :-)

Unfortunately, I cannot reproduce the problem. Can you send the data and the options defined to produce the graph (eventually you can send it to my mail address) ?

Thanks for using ChartNew.js
François

**Comment by GalakFayyar** [comment]
Hi Francois,

Thank you for your answer.
I took the last version of your script ChartNew.js and the problem appears to be solved ! I noticed you added some function in it and that do the job. 
I had this issue with the release of January of the CharNew.js file, but it seems you modified it recently, and I didn't noticed.
I suggest you to add a date of release on the header section of you file. If should help to quickly view the difference between versions.
Anyway, all is solved for me, all the values appear correctly for horizontal stacked bars even the zeros! 
Thanks for your work.

Regards,
G. Fayyar.
